content/github/building-a-strong-community/manually-creating-a-single-issue-template-for-your-repository.md

```diff
@@ -28,7 +28,7 @@ assignees: octocat
 ```
 {% note %}
 
-**Note:** You must put YAML-reserved characters such as `:` in quotes. For example, `":bug: Bug"`.
+**Note:** You must put YAML-reserved characters such as `:` in quotes. For example, `":bug: Bug"` or `":new: triage needed, :bug: bug`". Note that labels are not escaped individually.
```

Comment:
Your new example looks great, but I think that it might be good to update the original text too. Would this work?

```suggestion
**Note:** If a front matter value includes a YAML-reserved character such as `:` , you must put the whole value in quotes. For example, `":bug: Bug"` or `":new: triage needed, :bug: bug"`.
```

Reply:
I am absolutely fine with your changes. Feel free to update the text as you suggest. :-)